registry/donor/views.py

```diff
@@ -172,3 +181,29 @@ def save_note():
     db.session.commit()
     flash("Poznámka uložena.", "success")
     return redirect(url_for("donor.detail", rc=note_form.rodne_cislo.data))
+
+
+@blueprint.post("/override")
+@login_required
+def save_override():
+    override_form = DonorsOverrideForm()
+    delete = True if "delete_btn" in request.form else False
+
+    if override_form.validate_on_submit():
+        if not delete:
+            override = DonorsOverride(**override_form.field_data)
```

Comment:
Tady je potřeba ošetřit, zda náhodou override již pro dárce neexistuje, protože když se pokusím přeuložit existující, pokusí se to uložit jako nový záznam, což selže.

Reply:
Opraveno.